packages/@okta/vuepress-site/docs/guides/customize-tokens-returned-from-okta/add-groups-claim-dynamic/index.md

```diff
@@ -11,6 +11,8 @@ You can use this function anywhere to get a list of Groups of which the current
 | pattern            | Search term                                   | FALSE          | `"Eastern-Region"`, `"Eastern"`, `"-Region"`|
 | limit              | Maximum number of Groups returned. Must be a valid EL expression and evaluate to a value between 1 to 100. | FALSE | `1`, `50`, `100`|
 
+> **Note:** When you use `Groups.startWith`, `Groups.endsWith`, or `Groups.contains`, the `pattern` argument is matched and populated on the `name` attribute rather than a user's email. If you are targeting groups that may have duplicate group names (such as Google Groups), use the `getFilteredGroups` Group function instead. See the **Parameter Examples** section of [Use group functions for static group whitelists](/docs/guides/customize-tokens-returned-from-okta/static-whitelist/#use-group-functions-for-static-group-whitelists) for more information on the parameters used in this Group function.
```

Comment:
nit:
> ... populated on the `name` attribute rather than a user's email.
Do we want to say `group's email` instead?

Reply:
Also, instead of a `Note` could we have a `Warning`? I'd like the security implications of the `Groups.*` EL functions to be more explicit.

Reply:
@ronwaisberg-okta 

We don't use Warning notes in content, typically. The use of Warning is saved for specific Warning notifications in the UI. Would "Important" be ok?